application/controllers/Form_result.php

```diff
@@ -0,0 +1,9 @@
+<?php
+class Form_result extends CI_Controller
```

Comment:
Can we please change the name of this to something more meaningful, please.

Reply:
What do you suggest ? I had no inspiration

Reply:
This is a very general page that simply displays the message passed to it as parameter, so I gave it a general name

Reply:
maybe something like "DisplayMessage" ?

Reply:
DisplayMessage seems better:)

Reply:
Finally I chose "Info_message"
BTW, I also made the strings of Messages.php translatable